src/Dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -1,48 +1,55 @@
+import { useEffect, useState } from "react";
 import Header from "../Header/Header";
+import axios from "axios";
+import { useUser } from "../../../context/AuthContext";
+import DonutChart from "react-donut-chart";
 
 export default function Dasxboard() {
+  const { Token } = useUser();
+  const [todoCount, settodoCount] = useState<number>(0);
+  const [progressCount, setprogressCount] = useState<number>(0);
+  const [doneCount, setdoneCount] = useState<number>(0);
+  const [activatedEmployeeCount, setactivatedEmployeeCount] = useState<number>(0);
+  const [deactivatedEmployeeCount, setdeactivatedEmployeeCount] = useState<number>(0);
+
+  const getTasksCount = async () => {
+    try {
+      let response = await axios.get(
+        `https://upskilling-egypt.com:3003/api/v1/Task/count`,
+        { headers: { Authorization: `Bearer ${Token}` } }
+      );
+      settodoCount(response.data.done);
+      setprogressCount(response.data.inProgress);
+      setdoneCount(response.data.toDo);
+      // console.log (response.data.done);
+    } catch (error) {}
+  };
+  const getUserCount = async () => {
+    try {
+      let response = await axios.get(
+        `https://upskilling-egypt.com:3003/api/v1/Users/count`,
+        { headers: { Authorization: `Bearer ${Token}` } }
+      );
+      setactivatedEmployeeCount(response.data.activatedEmployeeCount );
+      setdeactivatedEmployeeCount(response.data.deactivatedEmployeeCount);
+     
+    } catch (error) {}
+  };
+
+  useEffect(() => {
+    getTasksCount();
+    getUserCount();
```

Comment:
This API should be called conditionally based on user role, since it can not be accessed from employee users.